funcs.sh

```diff
@@ -68,12 +68,12 @@ function composer_build {
 	composer run-script cloud-build
 }
 
-function set_node_version {
+function node_install {
 	. /root/.nvm/nvm.sh --no-use
 
 	if [[ -f ".nvmrc" ]]; then
-		echo "nvm use"
-		nvm use
+		echo "nvm install"
```

Comment:
THIS IS A POC - BUT

Maybe specify the version being installed as could help with debugging

Reply:
When you run `nvm install` it installs the version defined in `.nvmrc` file in the project itself. I believe the `nvm install` command outputs the version it is installing as part of the install process. But if not, we could maybe `cat` the `.nvmrc` file beforehand.

Reply:
This is an example of how it shows what version is being installed and used.
```
Running cloud-build...
nvm install
Found '/app/.nvmrc' with version <22>
Downloading and installing node v22.14.0...
Downloading https://nodejs.org/dist/v22.14.0/node-v22.14.0-linux-x64.tar.xz...
#################################################################################################################################################################### 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v22.14.0 (npm v10.9.2)
```